spec/content/manifest.spec.yml

```
@@ -13,11 +13,24 @@ spec:
       type: object
       additionalProperties: false
       properties:
+        datasets:
+          description: Description of the datasets this package can be used with.
+          type: array
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              name:
+                description: Name of the dataset.
+                type: string
+            required:
+              - name
         fields:
           description: Description of the fields this package can be used with.
```

Btw, maybe we can extend the descriptions of the `datasets` and `fields`, so it is less confusing how queries are expected to match.
```suggestion
          description: List of fields this package expects to find in an index. For a package to be used with an index, the index should contain all the fields listed here.
```